### DISCUSSION TITLE
[操作事故] PCL2 Plaza OSS 服务 is Down

### DISCUSSION THREAD

**Discussion by wuliaodexiaoluo** [Meloong-Git/PCL#5145] [讨论]
因为操作失误误将 WAF 执行了一遍全新安装，导致[服务](http://pclhomeplazaoss.lingyunawa.top:27000/)暂时不可用
请勿将此问题作为 PCL2 Bug 报告给开发者

此问题已上报维护，等待解决

预计受到影响的服务:
PCL2 Plaza OSS

受此事故影响的主页:
PCL2主页 杂志主页
PCL2主页 新闻速报
PCL2主页 PCL2 书库

真的很抱歉
（少女道歉中.jpg）(没有这个表情包，只能麻烦各位自行脑补下（（（)

**Comment by LinQingYuu** [comment]
>因为操作失误误将 WAF 执行了一遍全新安装，导致[服务](http://pclhomeplazaoss.lingyunawa.top:27000/)暂时不可用

>通常只能给小洛一份测试环境，因为正式环境容易被小洛玩炸    ----苏灵

**Comment by Joker2184** [comment]
WAF...得凌云来 

目前还没正式接入PCL....

小问题 看看今天凌云有没有空.jpg

**Reply by Joker2184** [reply]
@JingHai-Lingyun

**Comment by Joker2184** [comment]
> 因为操作失误误将 WAF 执行了一遍全新安装，导致[服务](http://pclhomeplazaoss.lingyunawa.top:27000/)暂时不可用 请勿将此问题作为 PCL2 Bug 报告给开发者
> 
> 此问题已上报维护，等待解决
> 
> 预计受到影响的服务: PCL2 Plaza OSS
> 
> 受此事故影响的主页: PCL2主页 杂志主页 PCL2主页 新闻速报 PCL2主页 PCL2 书库
> 
> 真的很抱歉 （少女道歉中.jpg）(没有这个表情包，只能麻烦各位自行脑补下（（（)

书库..跑路了.jpg 没搞

**Comment by CreeperIsASpy** [comment]
我上早八。。。

**Reply by Joker2184** [reply]
在修了.jpg

大概应该好了等我加白名单

**Comment by Joker2184** [comment]
修完了 链接正常 

http://pclhomeplazaoss.lingyunawa.top:27000/d/Homepages/Joker2184/UpdateHomepage.xaml 可以访问测试

直接访问27000理论是告警

PCL可下载

**Comment by MoYuan-CN** [comment]
事故结束，目前 HomePlaza OSS 服务（SSL）-> ✅ 正常
